data_retriever.py

```diff
@@ -1,25 +1,45 @@
 import requests
+import csv
 import json
 
-BASE_URL = "https://fantasy.premierleague.com/drf"
-STATIC = BASE_URL + "/bootstrap-static"
-FIXTURES = BASE_URL + "/fixtures"
-PLAYER = BASE_URL + "/element-summary/"
+player_teammates_2018_stats = {}
+
+PREVIOUS_SEASONS = {"2017-18": {}, "2016-17": {}}
+player_stats_dict = {"2018-19": player_teammates_2018_stats}
+player_stats_dict.update(PREVIOUS_SEASONS)
+
+
+def write_as_json(player_stats_season_dict):
+    filename_format = './json_data/stats_%s.json'
+    for season in player_stats_season_dict:
+        filename = filename_format % season
+        with open(filename, 'w') as jsonFile:
+            json.dump(player_stats_season_dict[season], jsonFile, indent=4, sort_keys=True)
+
+
+def create_dict_from_csv(season, player_name, player_id):
+    players_hist_stats = requests.get(GITHUB_STATS_BASE_URL % (season, playerName)).content.decode('utf-8')
+    dict_after_csv = csv.DictReader(players_hist_stats.splitlines(), delimiter=',')
+    player_stats_dict[season][player_id] = {NAME: player_name, STATS: list(dict_after_csv)}
+
+
+BASE_URL = "https://fantasy.premierleague.com/drf/%s"
+STATIC = BASE_URL % "/bootstrap-static"
+FIXTURES = BASE_URL % "/fixtures"
+PLAYER = BASE_URL % "/element-summary/"
 NAME = "name"
 STATS = "stats"
 
+GITHUB_STATS_BASE_URL = "https://raw.githubusercontent.com/vaastav/Fantasy-Premier-League/master/data/%s/players/%s/gw.csv?datatype=csv"
 ID_ELEMENT = 'id'
 
 playerToRunId = 253
```

Comment:
Do not see any use for this variable. Remove it as it can cause confusion with the PLAYER_A_ID variable in json_load.py

Reply:
that is used for getting the team data can't remove it, will have to keep in sync with the the one in json.py